testing/tests/DevExpress.ui/collectionWidgetParts/editingTests.js

```diff
@@ -2128,6 +2162,27 @@ module('reordering of items', () => {
         instance.reorderItem(item(0), item(1));
     });
 
+    test('itemReordered event should be fired if items reordered', function(assert) {
+        const items = [{ a: 0 }, { a: 1 }];
+
+        const $element = $('#cmp');
+        const instance = new TestComponent($element, {
+            items: items
+        });
+
+        instance.on('itemReordered', (args) => {
+            assert.equal($(args.itemElement).get(0), item(1), 'correct item element');
+            assert.equal(args.fromIndex, 0, 'correct from index');
+            assert.equal(args.toIndex, 1, 'correct to index');
+        });
+
+        const item = function(index) {
+            return instance.itemElements().eq(index).get(0);
+        };
```

Comment:
We have many test cases when we use this construction:
```
instance.itemElements().eq(X);
```

Maybe it would be good to share this function for all of them?